### DISCUSSION TITLE
What is the software shown in readme preview image

### DISCUSSION THREAD

**Discussion by MarcusPierce** [iptv-org/community#410] [Q&A] [answered]
kinda curious, seems like a macOS software.

**Comment by johnbrown51** [comment, marked as answer]
It's an older version of VLC I'm pretty sure

**Reply by ghost** [reply]
I'm using the current version of VLC and that screenshot looks the same as what I have

**Reply by thedavidweng** [reply]
thank you

**Comment by BellezaEmporium** [comment]
It is VLC. Arhey has got a Mac, hence the fact that pictures are Mac-ish looking.
